src/framework/xr/xr-hit-test-source.js

```diff
@@ -132,23 +133,46 @@ class XrHitTestSource extends EventHandler {
      * @private
      */
     updateHitResults(results, inputSource) {
+        if (inputSource && !inputSource.hitTestSourcesSet.has(this))
+            return;
+
+        let origin = poolVec3.pop();
+        if (!origin) origin = new Vec3();
```

Comment:
```suggestion
        const origin = poolVec3.pop() ?? new Vec3();
```

Reply:
I guess this is the conflict you need to resolve, this is in the other PR already?